packages/svelte/src/compiler/phases/2-analyze/index.js

```diff
@@ -478,7 +458,7 @@ export function analyze_component(root, source, options) {
 							// bind:this doesn't need to be a state reference if it will never change
 							if (
 								type === 'BindDirective' &&
-								/** @type {BindDirective} */ (path[i]).name === 'this'
+								/** @type {import('#compiler').BindDirective} */ (path[i]).name === 'this'
```

Comment:
Make this a import jsdoc at the type to get rid of the `import(...)` here?

Reply:
not possibly unfortunately, as `BindDirective` is already imported. we need the qualifier